### PR TITLE
config: Added SERVER variable

### LINKED ISSUE #60: config: opendata.cern.ch
The magic string `opendata.cern.ch` is hard-coded in many places in the client.

It would be good to create `config.py` to centralise the server location.

Example: see `reana-client` and e.g. `TIMECHECK` value set up there.

### PR DESCRIPTION
**Pull request by ParthS007**
closes #60

### CODE REVIEW

File: cernopendata_client/cli.py [hunk 1]
```diff
@@ -127,7 +128,7 @@ def get_metadata(server, recid, doi, title, output_value):
 )
 @click.option(
     "--server",
-    default="http://opendata.cern.ch",
+    default=SERVER,
     type=click.STRING,
     help="Which CERN Open Data server to query? [default=http://opendata.cern.ch]",
```
Comment (tiborsimko) [inline cernopendata_client/cli.py:133]: in the help as well, you can do stuff like:

```python
help="Which CERN Open Data server to query? [default={}]".format(SERVER),
```
```

File: cernopendata_client/config.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# cernopendata-client is free software; you can redistribute it and/or modify
+# it under the terms of the GPLv3 license; see LICENSE file for more details.
+
+"""cernopendata-client configuration."""
+
+import os
```
Comment (tiborsimko) [inline cernopendata_client/config.py:11]: `import os` needed?

File: cernopendata_client/config.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+"""Default CERN Open Data query server"""
```
Comment (tiborsimko) [inline cernopendata_client/config.py:13]: Two things:

- The docstring comes after the variable, not before it.
- Please amend the text to "Default CERN Open Data portal instance to query."

File: cernopendata_client/searcher.py [hunk 4]
```diff
@@ -218,13 +219,12 @@ def get_file_info_remote(recid, filtered_files=None):
...
     """
-    server = "http://opendata.cern.ch"
     file_info_remote = []
-    record_json = get_record_as_json(server=server, recid=recid)
+    record_json = get_record_as_json(server=SERVER, recid=recid)
     for file_info in record_json["metadata"]["files"]:
         file_checksum = file_info["checksum"]
         file_size = file_info["size"]
-        file_uri = file_info["uri"].replace("root://eospublic.cern.ch/", server)
+        file_uri = file_info["uri"].replace("root://eospublic.cern.ch/", SERVER)
```
Comment (tiborsimko) [inline cernopendata_client/searcher.py:227]: This is another nice candidate for centralising in `config.py`, perhaps under the name EOSPUBLIC_ROOT_URI... Or we could have SERVER_HTTP_URI (currently named SERVER) and SERVER_ROOT_URI (for `root:/...`) for consistency.

File: cernopendata_client/searcher.py [hunk 5]
```diff
@@ -16,6 +16,7 @@
 
 import click
 import requests
+from .config import SERVER
```
Comment (tiborsimko) [inline cernopendata_client/searcher.py:19]: I believe having a blank line before the dot import is the usual convention.

File: cernopendata_client/searcher.py (hunk 4 above)
Comment (ParthS007) [reply cernopendata_client/searcher.py:227]: Yeah, this sounds good 👍

File: cernopendata_client/cli.py (hunk 1 above)
Comment (ParthS007) [reply cernopendata_client/cli.py:133]: done 👍

File: cernopendata_client/config.py (hunk 3 above)
Comment (ParthS007) [reply cernopendata_client/config.py:13]: Done 👍

File: cernopendata_client/searcher.py (hunk 4 above)
Comment (ParthS007) [reply cernopendata_client/searcher.py:227]: what should be the `doc string` for `SERVER_ROOT_URI`?

File: cernopendata_client/searcher.py (hunk 5 above)
Comment (ParthS007) [reply cernopendata_client/searcher.py:19]: Done 👍

File: cernopendata_client/searcher.py (hunk 4 above)
Comment (tiborsimko) [reply cernopendata_client/searcher.py:227]: Something harmonious with the HTTP one, for example:

```python
SERVER_HTTP_URI = "http://opendata.cern.ch"
"""Default CERN Open Data server to query over HTTP protocol."""

SERVER_ROOT_URI = "root://eospublic.cern.ch"
"""Default CERN Open Data server to query over XRootD protocol."""
```

Note also trailing slash difference in the `replace` command, the XRootD protocol needs two slashes, so please beware to keep it there.

File: cernopendata_client/config.py (hunk 2 above)
Comment (ParthS007) [reply cernopendata_client/config.py]: Removed 👍

File: cernopendata_client/searcher.py (hunk 4 above)
Comment (ParthS007) [reply cernopendata_client/searcher.py]: Done 👍

**Pull request merged**